Cargo.toml

```diff
@@ -37,3 +37,6 @@ regex = "0.2"
 lazy_static = "1.0.0"
 flate2 = "1.0"
 rayon = "1.0.1"
+
```

Comment:
you need to update the cosmogony version number (0.3.1 is enough I think as this should not break retrocompatibility)

Reply:
done